docs/software.md

```diff
@@ -2,12 +2,13 @@
 
 ## Overview
 
-**DietPi-Software** is a core DietPi tool, allowing you to install **Ready to run** and **Optimized software** for your device. It automates the installation and configuration process for you and also applies specific optimizations unique to your hardware model. As a result DietPi-Software ensure you get the **maximum performance** from your device.
+**DietPi-Software** is a core DietPi tool, allowing you to install **Ready to run** and **Optimized software items** for your device. It automates the installation and configuration process for you and also applies specific optimizations unique to your hardware model. As a result DietPi-Software ensure you get the **maximum performance** from your device.
 
-??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
-    To install any of the **DietPi Optimised Software** listed below run from the command line `dietpi-launcher` and selected *DietPi-Software* or launch the tool `dietpi-software`.
+??? info "How do I run **DietPi-Software** and install **Optimised software** items?"
```

Comment:
Suggestion here and in similar cases below:
```
How do I run **DietPi-Software** to install pre-configured software?
```
or "optimised software", but without the capital `O` and `**` as this is not literally found in the script anymore.

Reply:
Thx. Changed to "optimized".